log4j-jul/src/site/markdown/index.md

```diff
@@ -74,3 +74,29 @@ Java Level | Log4j Level
 [`FINER`](http://docs.oracle.com/javase/6/docs/api/java/util/logging/Level.html#FINER) | `TRACE`
 [`FINEST`](http://docs.oracle.com/javase/6/docs/api/java/util/logging/Level.html#FINEST) | [`FINEST`](apidocs/org/apache/logging/log4j/jul/LevelTranslator.html#FINEST)
 [`ALL`](http://docs.oracle.com/javase/6/docs/api/java/util/logging/Level.html#ALL) | `ALL`
+
+
+# Log4j JDK Logging Bridge
+
+The LogManager is not always useable because you have to set a JVM wide effective system
+property - e.g. in web servers this is not possible if you are not the administrator.
+
+The [`Log4jBridgeHandler`](apidocs/org/apache/logging/log4j/jul/Log4jBridgeHandler.html) is an
+alternative that can be declaratively used via `logging.properties`.
+It is less performant than the LogManager but still okay to use.
```

Comment:
This should also note that logging methods are subject to filtering by BOTH java.util.logging and log4j 2.